### DISCUSSION TITLE
How to Get Error Reports When PDF++ Frequently Causes Obsidian to White Screen?

### DISCUSSION THREAD

**Discussion by CamWam** [RyotaUshio/obsidian-pdf-plus#300] [Q&A]
Currently, my environment is as follows: 
SYSTEM INFO:
	Obsidian version: v1.7.7
	Installer version: v1.7.7
	Operating system: Windows 10 Pro for Workstations 10.0.19045
	Login status: logged in
	Language: zh
	Catalyst license: insider
	Insider build toggle: on
	Live preview: on
	Base theme: dark
	Community theme: Blue Topaz v2024110601
	Snippets enabled: 20
	Restricted mode: off
	Plugins installed: 77
	Plugins enabled: 22
		1: Search In Canvas v1.1.1
		2: Git v2.28.2
		3: Recent Files v1.7.1
		4: Settings Search v1.3.10
		5: Version History Diff v2.2.1
		6: Chronology v1.1.11
		7: Excalidraw v2.6.7
		8: Canvas Block Reference v0.0.4
		9: Style Settings v1.0.9
		10: Easy Typing v5.5.8
		11: Remotely Save v0.5.25
		12: List Callouts v1.2.9
		13: Canvas2Document v1.2.3
		14: Update time on edit v2.4.0
		15: Multi Properties v1.1.2
		16: Latex Suite v1.9.7
		17: Remember File State v1.1.2
		18: Calendar v1.5.10
		19: Daily Notes Editor v0.1.8
		20: Edit History v0.1.3
		21: File Tree Alternative v2.6.0
		22: PDF++ v0.40.13

I often open many windows to use Obsidian. When using PDF++,(only one tab opens a pdf) I frequently encounter all screens turning white and cannot obtain report information through the developer tools console.

The console tells me it cannot read the page, with the message: “DevTools disconnected from the webpage. DevTools will automatically reconnect when the page reloads.” Previous records contain no valuable information. At this point, I notice that Obsidian’s memory usage is very low and CPU usage is almost zero. Waiting for a long time does not restore anything, and I have to close and restart. This situation also occurs when adding new colors

After restarting, I find that the software interface settings (such as the zoom level) revert to 100%.

Normal software usage
![正常情况](https://github.com/user-attachments/assets/5eeff917-4e0a-44d4-a4f2-596075ab595d)

When the white screen crash occurs (I have multiple windows open)
![微信截图_20241123114845](https://github.com/user-attachments/assets/6cdad0dd-97eb-428e-b3fe-7d536561300f)
![微信截图_20241123115045](https://github.com/user-attachments/assets/3e56b952-f2ac-4bd2-99d9-6efee74efe32)

**Comment by RyotaUshio** [comment]
Thanks for the report. This is the first time I receive such a report.
Seeing your debug info, there is many plugins operating at the same time.
First of all, I have to make sure that this problem is caused by PDF++, and not by Obsidian itself or other plugins.

Can you reproduce it in the sandbox vault (or a clean new vault) where PDF++ is the only plugin installed?
Does the problem disappear when you disable PDF++ and go through the same steps?
If so, please file a bug report with PDF++'s debug info attached. Error message is not mandatory so it's okay if you can't see it.

If a particular PDF file is necessary to reproduce the issue, please attach the file as well.

**Reply by CamWam** [reply]
Thank you for taking the time to respond. I suspect it might be related to opening many files and Windows windows, or perhaps a conflict with a certain plugin. I’ll try to look into it over the next few days. I think it’s very likely a conflict with the LaTeX Suite, as I often encounter a white screen when inputting LaTeX. However, this isn’t certain, as inputting the same content doesn’t always cause a crash...
